src/routes/projects/past.svelte

```diff
@@ -27,10 +26,10 @@
   export let projects;
   export let account;
   export let pages;
-  let currentPage = 1;
+  const currentPage = 1;
```

Comment:
How would the **current** page get updated if it is a `const`? The pagination component has been updated to react (bind) to `currentPage`